DXRMapFixups/DeusEx/Classes/DXRFixupM09.uc

```diff
@@ -326,6 +326,10 @@ function PreFirstEntryMapFixes()
         hoverHint = class'DXRTeleporterHoverHint'.static.Create(self, "", jock.Location, jock.CollisionRadius+5, jock.CollisionHeight+5, exit);
         hoverHint.SetBaseActor(jock);
 
+        if (#defined(vanilla)) {
+            dxr.dxInfo.startupMessage[0] = "New York City, Lower East Side Cemetery"; // fix "cemetery" misspelling
```

Comment:
maybe we should check the existing value before overwriting it

Reply:
I figured a check for vanilla maps was sufficient but it couldn't hurt.

Reply:
Technically this is more restrictive than a check for vanilla maps, since this is only if you are actually playing Vanilla.  Theoretically this could be applied to Revision using vanilla maps also (but probably not necessary)

Reply:
You're right. I guess it's still good to add an additional condition for localization purposes.